src/util/glob.ts

```diff
@@ -131,8 +144,14 @@ export async function glob(
   // We want each group to include all subsequent negated patterns. The simplest
   // way to do that is to build the groups backwards.
   for (let i = expandedPatterns.length - 1; i >= 0; i--) {
-    const pattern = expandedPatterns[i];
+    let pattern = expandedPatterns[i];
     const isExclusive = pattern[0] === '!';
+    if (isExclusive) {
+      pattern = pattern.slice(1); // Remove the "!"
+    }
+    if (opts.rerootToCwd) {
+      pattern = pattern.replace(/^\/+/, '');
```

Comment:
Is the `+` necessary in the regex here? I suppose it doesn't hurt to match extra if the user happens to put in something like `//foo`.

Reply:
Yeah, if the user did that (which is a weird but valid path) we actually _must_ remove both `//`s, or else we'd still interpret it relative to the filesystem root.

Reply:
Ah, okay did not know multiple slashes would be considered valid. TIL it's just treated like a single slash.

Reply:
Added a comment to clarify this.